src/ui/views/ViewUpdate.qml

```diff
@@ -195,7 +195,7 @@ VPNFlickable {
             anchors.horizontalCenter: undefined
             Layout.alignment: Qt.AlignHCenter
             onClicked: {
-                stackview.pop(StackView.Immediate);
```

Comment:
why this? in VPNAlert we call `stackview.push()`

Reply:
Yes, for the "update Recommended" case when the user clicks on the notification. 
If i understand main.qml right , we're pushing this to the VPNStackView#mainStackView if we do `MozillaVPN::instance()->controller()->updateRequired();` right? 
```
main.qml:136
                State {
                    name: VPN.StateUpdateRequired

                    PropertyChanges {
                        target: loader
                        source: "states/StateUpdateRequired.qml"
                    }},
```

Reply:
I think you can simply remove this line. We show the 'logout' button only when the update is required:
https://github.com/mozilla-mobile/mozilla-vpn-client/blob/main/src/ui/views/ViewUpdate.qml#L46-L70

and that happens if the ViewUpdate.qml is loaded from the main.qml directly:
main.qml -> state: VPN.StateUpdateRequired -> states/SateUpdateRequired.qml -> views/ViewUpdate.qml

so there is no `stackView` involved at all.